sites/footers/includes/footer.html

```diff
@@ -1,29 +1,86 @@
 <footer id="wb-info">
+	<h2>{{ i18nText-footerSite }}</h2>
```

Comment:
Just be more explicit and use "wb-inv"
```suggestion
	<h2 class="wb-inv">{{ i18nText-footerSite }}</h2>
```